GitUI/Avatars/InitialsAvatarProvider.cs

```diff
@@ -18,12 +18,26 @@ public class InitialsAvatarProvider : IAvatarProvider
         {
             var (initials, hashCode) = GetInitialsAndHashCode(email, name);
 
-            var avatarColor = _avatarColors[Math.Abs(hashCode) % _avatarColors.Length];
+            var avatarColor = _avatarColors[hashCode];
             var avatar = DrawText(initials, avatarColor, imageSize);
 
             return Task.FromResult<Image?>(avatar);
         }
 
+        private int GetDeterministicHashCode(string str)
```

Comment:
It'd be good to have some comments explaining what's going on

Reply:
I wrote something....